disnake/ext/commands/context.py

```diff
@@ -324,7 +324,7 @@ async def send_help(self, *args: Any) -> Any:
         from .core import Command, Group, wrap_callback
         from .errors import CommandError
 
-        bot = self.bot
+        bot: Union[Bot, AutoShardedBot] = self.bot
```

Comment:
sneaky change

Reply:
I was looking for occurrences of `help_command` and noticed that this entire method was mostly `Unknown`, but apparently pylance had just given up ~~for the umpteenth time today~~. Reverted in https://github.com/DisnakeDev/disnake/pull/849/commits/7b53fb022ddf31956867a9cdc190e6f0749b7ea2